### PR TITLE
Update libvirt_guest_installation.xml (PXE part))

### PR DESCRIPTION
**Pull request by aginies**
### PR creator: Description

Fix PXE enablement (there is no Network PXE but the NIC)

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* https://jira.suse.com/browse/DOCTEAM-1619

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP6/openSUSE Leap 15.6
  - [x] SLE 15 SP5/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2

- SLE 12
  - [ ] SLE 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [x] all necessary backports are done

### CODE REVIEW

File: xml/libvirt_guest_installation.xml [hunk 1]
```diff
@@ -302,15 +302,19 @@
         </step>
         <step>
           <para>
-            Under <guimenu>Enable boot menu</guimenu>, activate
-            <guimenu>Network PXE</guimenu> and confirm with
+            Under <guimenu>Enable boot menu</guimenu> and confirm with
```
Comment (sounix000) [inline xml/libvirt_guest_installation.xml:305]: It seems that you have deleted the text: "_, activate <guimenu>Network PXE</guimenu>_". I am very new to this guide as a maintainer, so I am not sure if the omission is intentional or accidental. But something seems off.

File: xml/libvirt_guest_installation.xml [hunk 2]
```diff
@@ -302,15 +302,19 @@
...
             <guimenu>Apply</guimenu>.
+            You can also find the NIC device in the boot Menu configuration and move
```
Comment (sounix000) [inline xml/libvirt_guest_installation.xml:307]: As a writer, I would like to add this information in a different way for ease of readability. Would you mind if I send a pull request to your branch?

File: xml/libvirt_guest_installation.xml (hunk 1 above)
Comment (aginies) [reply xml/libvirt_guest_installation.xml:305]: There is no more this option in virt-manager

File: xml/libvirt_guest_installation.xml (hunk 2 above)
Comment (aginies) [reply xml/libvirt_guest_installation.xml:307]: of course.

File: xml/libvirt_guest_installation.xml [hunk 3]
```diff
@@ -592,9 +613,15 @@ network=vnet_nated</screen>
...
         must be explicitly specified. Users can reference the JSON descriptor
-        files to find a firmware that satisfies their requirements. 
+        files to find a firmware that satisfies their requirements.
       </para>
-
+      <tip>
+        <para>
+          The directory <filename>/usr/share/qemu/firmware</filename> contains all the JSON files
+          used by &libvirt;. This file give you detailed information about the firmwares, including
+          the capabilitie of the features.
```
Comment (dariavladykina) [inline xml/libvirt_guest_installation.xml:622]:
```suggestion
          used by &libvirt;. This file gives you detailed information about the 
          firmwares, including the capabilities of the features.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some small corrections here. Thanks!

**Pull request merged**